LiskHQ/lisk-hub
# Fix a bug in showing vote summary - Closes #728

### What was the problem?
Explained in #728 

### How did I fix it?
I used the vote list to render the vote summary view. to do so, I had to use multiStep component
and create dedicated components for vote item and delegate item and hence, their dedicated list components. 
I've also cleaned up our unit tests.

### How to test it?
Make sure if you search for a delegate outside the scope of loaded delegates, and click next, you'll see all the delegates you votes for, in vote summary list.

### Review checklist
- The PR solves #728 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
